generators/app/dotnetHelper.js

```diff
@@ -172,10 +172,6 @@ DotNetHelper.prototype.updateProjectJson = function (cb) {
                 data.publishOptions.include.push('dockerfile.debug');
                 changed = true;
             }
-            if (data.publishOptions.include.indexOf('dockerfile.release') < 0) {
-                data.publishOptions.include.push('dockerfile.release');
-                changed = true;
-            }
```

Comment:
Why are you deleting this, we need this. But there was a merge issue with the rename and it should be Dockerfile